src/app/FakeLib/MSBuildHelper.fs

```diff
@@ -96,6 +96,15 @@ let msBuildExe =
             ]
             defaultArg (sources |> List.choose id |> List.tryHead) "xbuild"
         | false, _ ->
+            let getVSPathFromVSWhere ver =
+                ExecProcessAndReturnMessages(fun info ->
```

Comment:
I think this throws when the `VSWhere` file doesn't exist. Is this available on all VS Versions? What if no Visual Studio is installed?